examples/.jshintrc

```diff
@@ -76,7 +76,7 @@
     "mootools"      : false,    // MooTools
     "node"          : false,    // Node.js
     "nonstandard"   : false,    // Widely adopted globals (escape, unescape, etc)
-    "phantom"       : false,    // PhantomJS
+    "chrome"        : false,    // Chrome
```

Comment:
We should keep `phantom`, and we shouldn't add `chome`. Could you revert this change?